### DISCUSSION TITLE
How to install on windows

### DISCUSSION THREAD

**Discussion by Meruzzi** [liquidctl/liquidctl#580] [Q&A]
Hello guys, I trying to use the liquidctl but I'm don't know how to. I know there is a manual but english isn't my main language and I don't know anything of python, if someone cold make a video showing how to install or a manual with all the step by step. Thank you

**Comment by jonasmalacofilho** [comment]
Pelo seu perfil, parece que você fala português. Onde você está?

E se quiser me dê um ping no Discord que posso tentar ajudá-lo rapidamente.

**Reply by Meruzzi** [reply]
Sim, falo português. Eu moro no Brasil, RJ. 

Procurei o Discord no seu perfil mas não achei, se conseguir me chamar lá por favor: meruziz#5408

Obrigado!
